src/net.h

```diff
@@ -830,6 +837,7 @@ class CNode
             case ConnectionType::OUTBOUND_FULL_RELAY:
             case ConnectionType::BLOCK_RELAY:
             case ConnectionType::ADDR_FETCH:
+            case ConnectionType::PRIVATE_BROADCAST:
```

Comment:
Why do we want the private broadcast connections to have the desirable services? 
For example, if we are connecting to a peer with `NODE_NETWORK_LIMITED` and we are not close to the tip, in this case, the node wouldn't be desirable:

https://github.com/bitcoin/bitcoin/blob/41bf8f2d5eceaac72b98c1791f5fc368b3af90cb/src/net_processing.cpp#L1669-L1677

But what would be the problem? If the node only has to broadcast our transactions, why are we checking if it can serve past blocks to us?